15/umbraco-cms/fundamentals/data/defining-content/default-document-types.md

```diff
@@ -2,36 +2,40 @@
 
-## Document type
+## Element Type
 
-Creating a Document Type (without a template) is about defining the content structure and fields that can be used across different content items. You might use document types without templates for creating consistent, structured content that doesn't require a predefined page layout. For example blog posts or product listings.
+An Element Type is a Document Type without a template containing schema configurations for repeating a set of properties. These are for defining schema in the Block List Editor, Block Grid Editor, or other Element Type based editors.
 
-## Compositions
+Element Types cannot be used to create content that resides in the Content tree. When you create an Element type, it automatically sets the **Is Element Type** flag to **True** on the **Permissions** tab.
 
-Compositions provide a way to create reusable sets of properties that can be added to one or more document types. This can help simplify the management and consistency of content types across your website.
+![Element type](/15/umbraco-cms/fundamentals/data/images/element-type.png)
 
-When using a mixed setup, you can take advantage of nesting and use compositions by clicking on "**Compositions**..." option.
+Element Types are created using the same workflow as regular Document Types but usually contain fewer properties. You can also create Element Types as part of configuring a Block Grid or Block List Data Type.
 
-![Create group](/14/umbraco-cms/fundamentals/data/images/createGroup_new.png)
-{% hint style="warning" %}
+## Folder
 
-If you create 2 compositions that contain some common properties it is only possible to pick one of the compositions in a Document Type. If preferred, those compositions that cannot be used can be marked as hidden by checkmarking the `Hide unavailable options`.
+A Folder is a special type of Document Type that can be used to organize content in the Content tree. Folders can contain other content items, such as other folders or content nodes. They are useful for organizing content in a logical hierarchy, making it easier to manage and navigate your website's content. They cannot be used to create content that is displayed on the front end of your website.
 
-![Composition](/14/umbraco-cms/fundamentals/data/images/composition.png)
-{% endhint %}
+## Compositions
 
-## Element Type
+Compositions provide a way to create reusable sets of properties that can be added to one or more Document Types. This can help simplify the management and consistency of content types across your website. Compositions can be used to define common properties that are shared across multiple Document Types, such as metadata fields or social media links.
```

Comment:
```suggestion
Compositions provide a way to create reusable sets of properties that can be added to one or more Document Types. This can help simplify the management and consistency of content types across your website. Compositions can be used to define common properties shared across multiple Document Types, such as metadata fields or social media links.
```